scripts/generate-app-config/generateAppConfig.js

```diff
@@ -37,7 +37,7 @@ const addConfig = (config, fileName, buildPath, fs, imports) => {
  * } param0
  */
 const generateAppConfig = ({
-  siteConfig, mode, prefix, apps = [], verbose = false, basename, resolveExtensions,
+  siteConfig, mode, prefix, apps = [], verbose = false, basename, resolveExtensions = [],
```

Comment:
why was a default value added?

Reply:
To prevent the line below from blowing up. I noticed that it blew up when I ran the current `generateAppConfig` script.

https://github.com/cerner/terra-dev-site/blob/91f94d78827b226169aaf3e521066d04b8a33d3f/scripts/generate-app-config/generatePagesConfig.js#L143-L144

Reply:
honestly running that script outside of the webpack plugin probably isn't going to work right anymore.

Reply:
Sure, but irregardless of that, I think it's a good idea to default it to an empty array.